e2e_tests/README.md

```diff
@@ -1,40 +0,0 @@
-# E2E Tests
```

Comment:
I'm fine removing this copy of the manifests, but is it possible now to replicate CI behaviour/scripts locally? What about a readme file to describe how to do it?

Reply:
Yep, I'll add a readme